lxd/auth/drivers/openfga.go

```diff
@@ -256,11 +256,14 @@ func (e *embeddedOpenFGA) CheckPermission(ctx context.Context, entityURL *api.UR
 
 	// If not allowed, decide if the user can view the resource.
```

Comment:
Please can you tweak this comment as it reads a bit odd now:

> // If not allowed, decide if the user can view the resource.

This statement seems contradictory.

How about:

> // If not allowed, then decide if the user can view the resource in order to alter the HTTP response code.